Anlab.Mvc/Services/IOrderService.cs

```diff
@@ -16,22 +17,72 @@ public interface IOrderService
         Task PopulateOrder(OrderSaveModel model, Order orderToUpdate);
         void PopulateOrderWithLabDetails(OrderSaveModel model, Order orderToUpdate);
         Task SendOrderToAnlab(Order order);
+
+        List<TestItemModel> PopulateTestItemModel();
     }
 
     public class OrderService : IOrderService
     {
         private readonly ApplicationDbContext _context;
+        private readonly ITestItemPriceService _itemPriceService;
+        private readonly AppSettings _appSettings;
 
-        public OrderService(ApplicationDbContext context)
+        public OrderService(ApplicationDbContext context, ITestItemPriceService itemPriceService, IOptions<AppSettings> appSettings)
         {
             _context = context;
+            _itemPriceService = itemPriceService;
+            _appSettings = appSettings.Value;
+        }
+
+        public List<TestItemModel> PopulateTestItemModel()
+        {
+            var prices = _itemPriceService.GetPrices();

```

Comment:
@jSylvestre needs to change in the future to an async so might as well do it now and just return Task.Result() so we don't have a massive change of method signatures later on